forem/forem
# Reindex comments' Elasticsearch doc when commentable is updated

## What type of PR is this? (check all applicable)

## Description
- Comment's Elasticsearch doc currently has a publish status that defaults to `true`. This updates that to the status of the comment's commendable (ie. articles and podcasts).
- Unpublishing an article now reindexes its comment to reflect the publish status.

## Related Tickets & Documents
n/a

## QA Instructions, Screenshots, Recordings
0. Pull down this PR and startup local server _with Sidekiq and Elasticsearch_ running.
1. Create a brand new article, 
1. Add a unique comment to it.
1. Try searching for the comment, you should be able to see the search result.
1. Now unpublished the article you just created.
1. You should no longer be able to search for your old comment.  
## Added tests?
## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
Monitor the status of this DataUpdateScript
